src/components/storage/DiskEncryption.jsx

```diff
@@ -70,11 +70,17 @@ const DiskEncryption = ({
 }) => {
     const { partitioning } = useContext(StorageContext);
     const request = partitioning?.requests?.[0];
-    const [confirmPassword, setConfirmPassword] = useState(request?.passphrase || "");
-    const [isEncrypted, setIsEncrypted] = useState(request?.encrypted);
-    const [password, setPassword] = useState(request?.passphrase || "");
+    const [confirmPassword, setConfirmPassword] = useState();
+    const [isEncrypted, setIsEncrypted] = useState();
+    const [password, setPassword] = useState();
     const luksPolicy = useContext(RuntimeContext).passwordPolicies.luks;
 
+    useEffect(() => {
+        setIsEncrypted(request?.encrypted);
+        setPassword(request?.passphrase);
+        setConfirmPassword(request?.passphrase);
```

Comment:
I guess we are fine with not requiring string here (dropping the `|| ""`) part.

Reply:
I noticed that from the API these are empty strings by default.
https://github.com/rhinstaller/anaconda/blob/master/pyanaconda/modules/common/structures/partitioning.py#L38

so no need for this.
I will split this change is seperate commit for clarify, it's not nice that I hide this in this massive PR.

Reply:
Thank you, feel free to keep it, fine by me.

Reply:
Split